src/NuGet.Server/Infrastructure/WebConfigSettingsProvider.cs

```diff
@@ -1,18 +1,29 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
-using System;
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Specialized;
 using System.Web.Configuration;
 using NuGet.Server.Core.Infrastructure;
 
 namespace NuGet.Server.Infrastructure
 {
     public class WebConfigSettingsProvider : ISettingsProvider
     {
+        private readonly NameValueCollection _settings;
```

Comment:
Here and in the PackageAuthenticationService  there are two different patterns for initialization of the settings. Any reason for changing the pattern?